libdrgn/Makefile.am

```diff
@@ -17,7 +17,8 @@ include_HEADERS = drgn.h
 
 ARCH_DEFS_PYS = arch_aarch64_defs.py \
 		arch_ppc64_defs.py \
-		arch_x86_64_defs.py
+		arch_x86_64_defs.py \
+		arch_s390_defs.py
```

Comment:
I think you may have forgotten to `git add` `arch_s390_defs.py`, I don't see it here.

Also, would you mind putting s390 before x86_64 to keep this alphabetically sorted?

Reply:
Indeed. Sorry.